<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForTwoPOI</class>
 <widget class="QWidget" name="ForTwoPOI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>506</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>973</width>
    <height>506</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Compare two POI based on info</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Prepare</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>location_id 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>location_id 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>from</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dayfrom">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2009</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2010</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2009</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QDateEdit" name="dayto">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2010</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2010</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2009</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QSpinBox" name="poi1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="poi2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
  </widget>
  <widget class="QtCharts::QChartView" name="linechartView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>571</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="total1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total 1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="max1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max 1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="total2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total 2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="max2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max 2:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
